projects/plugins/jetpack/extensions/shared/jetpack-plugin-sidebar.js

```diff
@@ -16,6 +18,21 @@ const { Fill, Slot } = createSlotFill( 'JetpackPluginSidebar' );
 
 export { Fill as default };
 
+/**
+ * Open Jetpack plugin sidebar by default when URL includes jetpackSidebarIsOpen=true.
+ */
+function openJetpackSidebar() {
+	if ( getQueryArg( window.location.search, 'jetpackSidebarIsOpen' ) !== 'true' ) {
+		return;
+	}
+
+	dispatch( 'core/interface' ).enableComplementaryArea(
+		'core/edit-post',
+		'jetpack-sidebar/jetpack'
+	);
+}
+openJetpackSidebar();
```

Comment:
What do you think about wrapping this function with the dom ready function?

```suggestion
domReady( openJetpackSidebar );
```

Reply:
don't forget to import the lib

```es6
import domReady from '@wordpress/dom-ready';
```

Reply:
Thanks, that works! Changed here: https://github.com/Automattic/jetpack/pull/21156/commits/8309c23a1b9d6ddce85c124b297abf486dc9ecf4